browser-test/test.endtoend.ts

```diff
@@ -0,0 +1,546 @@
+import * as assert from 'assert';
+import { async } from 'walkdir';
+import {expect} from 'chai';
```

Comment:
Lines 2 and 3 are formatted differently, I'm guessing you still need to run `npx gts fix`.